modules/ROOT/pages/endpoints.adoc

```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-=======
+// ====
+
+== Delete Endpoints
+
+From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations.
+
+. From within your Anypoint Partner Manager Sandbox or Production environment, select Partner or <host partner name> from the upper left.
+
+. Search for the endpoint you want to delete by Name, Protocol, URL, or Description.
```

Comment:
Lowercase these as common nouns (you are searching for a name or protocol, for example, and not the label Name or the label Protocol).

Reply:
```suggestion
. Search for the endpoint you want to delete by name, protocol, URL, or description.
```

Reply:
Fixed.